src/Routes.test.js

```diff
@@ -0,0 +1,260 @@
+import ReactDOM from 'react-dom';
+import { mount } from 'enzyme';
+import { MemoryRouter } from 'react-router';
+import React from 'react';
+import Routes from './Routes';
+import App from './App';
+
+jest.setTimeout(1000);
+
+import Home from './pages/Home';
+import NotFound from './pages/NotFound';
+import SolicitaChaveAcesso from './pages/SolicitaChaveAcesso'
+import VerificaChaveAcesso from './pages/VerificaChaveAcesso'
+import Evento from './pages/Evento'
+import Participantes from './pages/Participantes'
+import Participante from './pages/Participante' 
+import CadastroConta from './pages/CadastroConta'
+import AcessoConta from './pages/AcessoConta'
+import Perfil from './pages/Perfil'
+import CadastroEvento from './pages/CadastroEvento'
+import PoliticaPrivacidade from './pages/PoliticaPrivacidade'
+import TermoUso from './pages/TermoUso'
+
+test('Rota para Home', (done) => {
```

Comment:
Uma dúvida, como tem vários casos de testes o ideal não seria envolvê-los dentro de um describe?

Reply:
O describe nem é tao necessário assim, não é uma rega um it ou varios its dentro de um describe

Reply:
Ah legal =D